src/ngraph/runtime/gpu/gpu_util.cpp

```diff
@@ -192,3 +192,62 @@ uint32_t runtime::gpu::idiv_ceil(int n, int d)
     // compiler fused modulo and division
     return n / d + (n % d > 0);
 }
```

Comment:
The gpu::stopwatch class crosses the nGraph - JIT CXX ABI boundary. The class will be compiled by the nGraph static compiler but declared in the JIT. This will be unsafe if the two compilers have different ABIs. For that reason, I would suggest making this a header-only class (and marking some of those functions as `inline` for guidance)

Reply:
Change to inline might increase compile time a lot. Make nGraph strict ABI compatible is a hard task.

Reply:
moved timer to runtime_context